gateway-provider-security-jwt/src/main/java/org/apache/knox/gateway/provider/federation/jwt/filter/JWTAuthCodeAssertionFilter.java

```diff
@@ -42,6 +43,7 @@ public class JWTAuthCodeAssertionFilter extends AbstractIdentityAssertionFilter
   private JWTokenAuthority authority;
 
   private ServiceRegistry sr;
+  private String tokenIssuer;
 
   @Override
   public void init( FilterConfig filterConfig ) throws ServletException {
```

Comment:
Not related to this change but we have some code duplication here with `JWTAccessTokenAssertionFilter` We can prolly combine it into a common base class. Not required here but would be nice.

Reply:
Yes, or maybe factor out some parts to some other classes and reference those from them. But this is probably out of scope of this change.